internal/pkg/cli/pipeline_init.go

```diff
@@ -192,6 +199,57 @@ func (o *initPipelineOpts) Ask() error {
 	return nil
 }
 
+// validateDuplicatePipeline checks that the pipeline name isn't already used
+// by another pipeline, whether it's been deployed or just has a local manifest file.
+// We check for the existance of the name and the namespaced name to reduce
+// potential confusion with a legacy pipeline.
+func (o *initPipelineOpts) validateDuplicatePipeline() error {
+	// make sure pipeline isn't already deployed
+	names, err := o.codePipeline.ListPipelineNamesByTags(map[string]string{
+		deploy.AppTagKey: o.appName,
+	})
+	if err != nil {
+		return fmt.Errorf("validate if pipeline exists: %w", err)
+	}
+
+	fullName := fmt.Sprintf(fmtPipelineName, o.appName, o.name)
+	for _, name := range names {
+		if strings.EqualFold(name, o.name) || strings.EqualFold(name, fullName) {
+			log.Errorf(`It seems like you are trying to init a pipeline that already exists.
+To recreate the pipeline, please run:
+1. %s. Note: The manifest file will not be deleted and will be used in Step 2.
+If you'd prefer a new default manifest, please manually delete the existing file.
+2. And then %s
```

Comment:
nit: can we add a verb after "then"? maybe "run"?

Reply:
see what you think about it now :)